### ISSUE TITLE
Show mentions dropdown only once

### ISSUE THREAD

**Issue by vanyasem** [hendraanggrian/socialview#51]
It would be nice to have a feature that would trigger that bug #12. I have a use case when you can only mention one person in a text field.

**Comment by vanyasem** [comment]
I made a hack that dismisses the dropdown if a mention is already present in `afterTextChanged`, but that introduces a regression, as if a user manually types the mention, it will still get colored (albeit this is somewhat softened by #52)

**Comment by hendraanggrian** [comment]
I find it similar to #52 in which both issues are about limitation of mentions.
